default.nix

```diff
@@ -54,6 +54,7 @@ in
           doCheck = runTests;
           doHaddock = runTests;
           doCoverage = runCoverage;
+          testTarget = "--test-option=--hide-successes";
```

Comment:
Which tool is expecting `--test-option`? Is it the underlying cabal?

Reply:
Nix passes it to `Setup.hs` under the hood.